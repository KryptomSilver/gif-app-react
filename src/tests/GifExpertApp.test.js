import React from "react";
const { shallow } = require("enzyme");
import GifExpertApp from "../GifExpertApp";

describe("Prubas en el componente <GifExpertApp/>", () => {
  test("Debe de mostrar el componente correctamente", () => {
    const wraper = shallow(<GifExpertApp />);
    expect(wraper).toMatchSnapshot();
  });
  test("Debe de mostrar la lista de categorias", () => {
    const categories = ["One Puch", "Dragon Ball"];
    const wraper = shallow(<GifExpertApp defaultCategories={categories} />);
    expect(wraper).toMatchSnapshot();
    expect(wraper.find("GifGrid").length).toBe(categories.length);
  });
});
