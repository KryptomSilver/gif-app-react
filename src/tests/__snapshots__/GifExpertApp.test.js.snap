// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prubas en el componente <GifExpertApp/> Debe de mostrar el componente correctamente 1`] = `ShallowWrapper {}`;

exports[`Prubas en el componente <GifExpertApp/> Debe de mostrar la lista de categorias 1`] = `ShallowWrapper {}`;
